lumigator/python/mzai/sdk/tests/data/job-all.json

```diff
@@ -0,0 +1,11 @@
+{
+    "name": "test-job-001",
+    "description": "This is a test job",
+    "model": "some_model",
+    "dataset": "6f6487ac-7170-4a11-af7a-0f6db1ec9a74",
+    "max_samples": 5,
+    "model_url": "s3://some_url",
+    "system_prompt": "some prompt",
+    "config_infer_template": "some_infer_template",
```

Comment:
I am not sure this should work, we are not using `config_*_template` and especially not both inference and eval together now